documentation/documentation.html

```diff
@@ -195,7 +195,7 @@
           <div id="typography-container"></div>
           <hr />
           <div id="toast-container"></div>
-          <hr /> -->
+          <hr />  --> 
```

Comment:
I would suggest you remove this code if it's not needed.

Reply:
removed it !!